src/db/models/profile.js

```diff
@@ -0,0 +1,13 @@
+module.exports = (sequelize, DataTypes) => {
+  const Profile = sequelize.define('Profile', {
+    userId: DataTypes.INTEGER,
+    firstName: DataTypes.STRING,
+    lastName: DataTypes.STRING,
+    bio: DataTypes.STRING,
+    imgURL: DataTypes.STRING
+  }, {});
+  Profile.associate = function(models) {
```

Comment:
You didn't associate `Profile` to `User` in any way.

Reply:
I thought it was not necessary ATM but I will do that